pygmt/base_plotting.py

```diff
@@ -606,47 +606,52 @@ def grdview(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
+        A="straight_lines",
         B="frame",
-        S="style",
+        C="cmap",
+        D="offset",
+        E="error_bars",
```

Comment:
`error_bars` or `error_bar`?

Reply:
Again, GMT.jl uses `error_bars`, but we should probably use (singular) `error_bar`.